### ISSUE TITLE
dependency for root build.gradle

### ISSUE THREAD

**Issue by Marchuck** [scana/ok-gradle#25]
hello 
thanks for the awesome library!

I have a question - does this library helps with suggesting which dependency comes from, I mean sometimes I also need to add to root build.gradle something like this:

```
allprojects {
		repositories {
			...
			maven { url 'https://jitpack.io' }
		}
	}
```
is there a plan for supporting it?

**Comment by scana** [comment]
Yeah - the plugin could verify whether there is a specific repository present already and suggest adding it when it is not. 

Awesome idea, I will look into it, thank you!
